src/funjson.c

```diff
@@ -64,6 +65,27 @@ json_escape_string(char *input)
   return buff;
 }
 
+/** Unescape a string for use as a JSON string. Returns a calloced buffer under 'json.string.partialbuf'. **/
+char * json_unescape_latin1string(char *latin1, int len)
+{
+  char *c;
+  char *partialbuf = mush_calloc(len*6,sizeof(char),"json.string.partialbuf");
```

Comment:
As far as I can see, the contents of partialbuf may be smaller or equal to latin1 but should never be longer, so I'm not sure this *6 is necessary? I'm wondering if you might actually be better off with a static char[BUFFER_LEN] here; in every instance I can see you're using the value once and then writing/copying it elsewhere and immediately freeing it, and I don't think you can have anything larger than BUFFER_LEN coming in from your usage?

I also can't see that the "len" arg to this function is ever actually used?

Reply:
To explain the *6 (which was high even for what I was looking to do), originally I was looking to add closing or opening markup where needed.  But this is no longer needed.